forest/src/cli/chain_cmd.rs

```diff
@@ -46,50 +47,36 @@ impl ChainCommands {
         match self {
             Self::Block { cid } => {
                 let cid: Cid = cid.parse().unwrap();
-                let mut client = new_client();
-
-                let blk = block(&mut client, cid)
-                    .await
-                    .map_err(stringify_rpc_err)
-                    .unwrap();
+                let blk = block(cid).await.map_err(stringify_rpc_err).unwrap();
                 println!("{}", serde_json::to_string_pretty(&blk).unwrap());
             }
             Self::Genesis => {
-                let mut client = new_client();
-
-                let gen = genesis(&mut client)
-                    .await
-                    .map_err(stringify_rpc_err)
-                    .unwrap();
+                let gen = genesis().await.map_err(stringify_rpc_err).unwrap();
                 println!("{}", serde_json::to_string_pretty(&gen).unwrap());
             }
             Self::Head => {
-                let mut client = new_client();
-
-                let canonical = head(&mut client).await.map_err(stringify_rpc_err).unwrap();
+                let canonical = head().await.map_err(stringify_rpc_err).unwrap();
                 println!(
                     "{}",
-                    serde_json::to_string_pretty(&canonical.0.cids()).unwrap()
+                    serde_json::to_string_pretty(
+                        &canonical
+                            .0
+                            .cids()
+                            .iter()
+                            .map(|cid: &Cid| base64::encode(&cid.to_bytes()))
```

Comment:
@ec2 buddy, this didn't work like I thought it would
I want proper cids, don't I?

To compare, I ran:

`FULLNODE_API_INFO="/ip4/127.0.0.1/tcp/1234/http" lotus chain head && target/release/forest chain head`

An example of what this branch of forest outputs:

```
[
  "AXGg5AIgos5/lOLLPc4KgPo+FspWt6zhQg1jc9z5bMJhBBkVqVY=",
  "AXGg5AIgP3+0yylaV5kSH/H63wUHhgDku+R1ByR7SZAk1XFNcMc=",
  "AXGg5AIgCIQ7YlecRZgMjjN9mp3AqonaSsepSW/nvi9NicRj7i4=",
  "AXGg5AIg8YnMkQAyXr5nOevGu98pUKHpjDCW/jDfrPVpCYGMBEk=",
  "AXGg5AIgnaTQgw3872sqgeDsYiNkVetgLqqEKn834Ldf28eL+IE=",
  "AXGg5AIgqsaPhPKrKKlgnX1QB/W+Mc8JvpoqtAn+Wc+oAb0cBZw=",
  "AXGg5AIgZVYwSQSQv3wi//FxeDN/YJuVfnPNnmSQyO8VfwM2wzY=",
  "AXGg5AIgZdvSYybA9iJir15ZHycp+HsUIOuUjKIa803CoW63KM0=",
  "AXGg5AIgrmI5G4UujZZJF/5a4eIbCFKsRrH896+HmkIzifErRgE="
]
```

Meanwhile lotus outputs:

```
bafy2bzacecrm474u4lft3tqkqd5d4fwkk232zykcbvrxhxhzntbgcbazcwuvm
bafy2bzacea7x7nglffnfpgisd7y7vxyfa6dabzf34r2qojd3jgicjvlrjvymo
bafy2bzaceaeiio3ck6oelgamryzx3gu5ycvitwsky6uus37hxyxu3coempxc4
bafy2bzacedyytteraazf5pthhhv4no67ffikd2mmgclp4mg7vt2wscmbrqces
bafy2bzaceco2juedbx6o62zkqhqoyyrdmrk6wybovkccu7zx4c3v7w6hrp4ic
bafy2bzacecvmnd4e6kvsrklatv6vab7vxyy46cn6tivlicp6lhh2qan5dqczy
bafy2bzacebsvmmcjasil67bc77yxc6btp5qjxfl6opgz4zeqzdxrk7ydg3btm
bafy2bzacebs5xutde3apmitcv5pfshzhfh4hwfba5okiziq26ng4filow4um2
bafy2bzacecxgeoi3quxi3fsjc77fvypcdmefflcgwh6ppl4htjbdhcprfndac
```

Reply:
Eric let me know via PM last night that Cid implements `to_string()`, so that works now.